IsraelHiking.Web/sources/application/components/sidebar/layersSidebar.css

```diff
@@ -4,8 +4,12 @@
     margin-bottom: 5px;
 }
 
-.row-hover:hover {
-    text-decoration: none;
-    color: black;
-    background: #bbb;
+.row-hover {
+    transition: background .2s ease;
 }
+
+    .row-hover:hover {
+        text-decoration: none;
+        color: black;
+        background: #bbb;
+    }
```

Comment:
I already moved some code around, assuming I understood what you wanted to do.
Please take my latest changed from yesterday.